core/backend/src/ChangeSummaryManager.ts

```diff
@@ -174,8 +175,9 @@ export class ChangeSummaryManager {
 
     // Detach change cache as it's being written to during the extraction
     const isChangeCacheAttached = this.isChangeCacheAttached(iModel);
-    if (isChangeCacheAttached)
-      ChangeSummaryManager.detachChangeCache(iModel);
+    if (isChangeCacheAttached) {
+      throw new IModelError(DbResult.BE_SQLITE_ERROR, "There is a attach change cache file. Re-open the connection to detach it.");

```

Comment:
Suggestion - "There is an attached change cache file."

Reply:
fixed